docs/pages/_error.js

```diff
@@ -221,7 +229,7 @@ function replaceVersionWithLatest(path) {
 }
 
 function pathRequiresVersioning(path) {
-  return path.match(new RegExp(SDK_PATH_PATTERN));
+  return path.match(new RegExp(SDK_PATH_PATTERN)) || path.match(new RegExp(REACT_NATIVE_PATH_PATTERN));
```

Comment:
@brentvatne, do we want to keep this or remove? I think it was obsolete after we changed the `/sdk/react-native` to a regex.

Reply:
i think we should keep it - it's accurate for the name of the function